WooCommerce/Classes/ViewRelated/Products/Cells/ProductsTabProductTableViewCell.swift

```diff
@@ -0,0 +1,89 @@
+import UIKit
+
+final class ProductsTabProductTableViewCell: UITableViewCell {
+    private lazy var productImageView: UIImageView = {
+        let image = UIImageView(frame: .zero)
+        return image
+    }()
+
+    private lazy var nameLabel: UILabel = {
+        let label = UILabel(frame: .zero)
+        return label
+    }()
+
+    private lazy var detailsLabel: UILabel = {
+        let label = UILabel(frame: .zero)
+        return label
+    }()
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        configureSubviews()
+        configureNameLabel()
+        configureDetailsLabel()
+        configureProductImageView()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+extension ProductsTabProductTableViewCell {
+    func update(viewModel: ProductsTabProductViewModel) {
+        nameLabel.text = viewModel.name
+
+        detailsLabel.attributedText = viewModel.detailsAttributedString
+
+        productImageView.contentMode = .center
+        productImageView.image = .productsTabProductCellPlaceholderImage
+        if let productURLString = viewModel.imageUrl {
+            productImageView.downloadImage(from: URL(string: productURLString), placeholderImage: nil, success: { [weak self] _ in
+                self?.productImageView.contentMode = .scaleAspectFill
+            })
+        }
+    }
+}
+
+private extension ProductsTabProductTableViewCell {
+    func configureSubviews() {
+        let contentStackView = createContentStackView()
+        let stackView = UIStackView(arrangedSubviews: [productImageView, contentStackView])
+        stackView.axis = .horizontal
+        stackView.spacing = 16
+        stackView.alignment = .leading
+
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        contentView.addSubview(stackView)
+        contentView.pinSubviewToAllEdges(stackView, insets: UIEdgeInsets(top: 16, left: 16, bottom: 16, right: 16))
+    }
+
+    func createContentStackView() -> UIView {
+        let contentStackView = UIStackView(arrangedSubviews: [nameLabel, detailsLabel])
+        contentStackView.translatesAutoresizingMaskIntoConstraints = false
+        contentStackView.axis = .vertical
+        return contentStackView
+    }
+
+    func configureNameLabel() {
+        nameLabel.applyBodyStyle()
```

Comment:
Would it make sense to make this configuration part of the closure that initialises the label? (as private lazy var)

Reply:
From my observation of the codebase it seems like we style each view in its own `configure*` function. I think it might be worth keeping the existing pattern styling views in the functions, and it could be easier to locate all the styling for one view as we discussed